elife/external-volume.sls

```diff
@@ -20,6 +20,20 @@ mount-point-external-volume:
     file.directory:
         - name: {{ pillar.elife.external_volume.directory }}
 
+mount-point-external-volume-existing-data-move-out:
+    cmd.run:
+        - name: |
+            touch {{ pillar.elife.external_volume.directory }}/ping
+            if test -e /etc/init.d/docker; then stop docker; status docker; fi
+            if systemctl is-enabled --quiet docker; then systemctl stop docker; fi
```

Comment:
```
root@heavybox--vagrant:/home/elife# systemctl is-enabled docker
enabled
root@heavybox--vagrant:/home/elife# systemctl stop docker
root@heavybox--vagrant:/home/elife# systemctl is-enabled docker
enabled
```

you might want 'is-active':

```
root@heavybox--vagrant:/home/elife# systemctl start docker
root@heavybox--vagrant:/home/elife# systemctl is-active docker
active
root@heavybox--vagrant:/home/elife# systemctl stop docker
root@heavybox--vagrant:/home/elife# systemctl is-active docker
inactive
```

Reply:
I started with that, but I wanted a command that communicated with an exit code rather than with output